scripts/build/termux_get_repo_files.sh

```diff
@@ -1,64 +1,84 @@
 termux_get_repo_files() {
+	local PACKAGES_HASH RELEASE_FILE repo_base dl_prefix RELEASE_FILE_URL RELEASE_FILE_SIG_URL pids=()
```

Comment:
The array should be more explicit.
```suggestion
	local PACKAGES_HASH RELEASE_FILE repo_base dl_prefix RELEASE_FILE_URL RELEASE_FILE_SIG_URL 
	local -a pids=()
```

Reply:
Done.